guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -74,6 +74,11 @@ ifdef::katello,orcharhino,satellite[]
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+2]
 endif::[]
 
+ifdef::katello,orcharhino,satellite[]
+//Migrating Linux 7 to Linux 8
+include::migrating_linux_7_to_linux_8.adoc[leveloffset=+2]
```

Comment:
@sabuchan07 I think I found the issue
```suggestion
include::topics/migrating_linux_7_to_linux_8.adoc[leveloffset=+2]
```

Reply:
I'd recommend that you try accepting the suggested change to the path. When you do this, you'll trigger a re-run of the automated checks, and with the path fixed and the include statement resolving correctly, your PR should pass all of them.